first_hour_on_terra/working_with_data_collections.ipynb

```diff
@@ -0,0 +1,449 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "from ipywidgets import Button, Layout\n",
+    "\n",
+    "CURRENT_WORKSPACE_ID = !terra workspace describe --format=json | jq --raw-output \".id\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Create a data collection\n",
+    "\n",
+    "The process of creating a data collection requires you to specify the following:\n",
+    "1. What data do you want to share? What type of resources--Cloud Storage buckets or objects, BigQuery tables, GitHUb repositories--will be made available via this data collection?\n",
+    "1. With whom do you wish to share this data? Will you be sharing the data collection with all members of an existing Terra group (e.g. for your organization or team), or will you need to create a new Terra group in order to restrict access to the data collection?\n",
+    "1. Will you update the data collection by releasing future versions?\n",
+    "\n",
+    "<div class=\"alert alert-block alert-success\">\n",
+    "<b>Note:</b> \n",
+    "    If you'd like to restrict access to your data collection to members of a specific group, you'll need to provide the <a href=\"https://et-docs-tests.googleplex.com/docs/reference/glossary/#policy\">group policy constraint</a> at the time of workspace creation. See <a href=\"../creating_a_group.ipynb\">../creating_a_group.ipynb</a> for details on how to create a Terra group that can be used for group policy constraints on workspaces and data collections. </div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Create a new workspace\n",
+    "\n",
+    "In order to create a data collection, you must first create a new workspace. Run the cell below to create a new workspace. The output should resemble:\n",
+    "\n",
+    "```\n",
+    "Workspace successfully created.\n",
+    "ID:                <GOOGLE_PROJECT_ID>-dc-ws\n",
+    "Name:              <TERRA_USER_EMAIL>-Data\n",
+    "Description:       A new workspace which I will transform into a data collection.\n",
+    "Cloud Platform:    GCP\n",
+    "Google project:    <GOOGLE_PROJECT_ID>\n",
+    "Cloud console:     https://console.cloud.google.com/home/dashboard?project=terra-vpp-arctic-mustard-2133\n",
+    "Properties:\n",
+    "  terra-type: workspace\n",
+    "Created:           YYYY-MM-DD\n",
+    "Last updated:      YYYY-MM-DD\n",
+    "# Resources:       0\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "input_style= {'description_width':'initial'}\n",
+    "buttonOutput = 'Please provide appropriate values in the input boxes.'\n",
+    "\n",
+    "input_workspace_id = widgets.Text(\n",
```

Comment:
Running this cell I get an import error. I think you're missing `from ipywidgets import widgets`?

Reply:
fixed!

Reply:
Instead of a new cell, could you please merge this with the other ipywdigets imports in the first cell?

Reply:
Can you double check your imports? For example I see you're using `Layout` and `os` without importing them, which causes import errors. What I'd suggest doing is clicking on kernel -> restart kernel and then running the notebook again from the top.

Reply:
I have refactored this and cleaned up the unnecessary imports.